usaspending_api/api_contracts/contracts/v2/agency/toptier_code/federal_account/count.md

```diff
@@ -0,0 +1,34 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Federal Account Count [/api/v2/agency/{toptier_code}/federal_account/count/{?fiscal_year}]
+
+Returns the count of unique Federal Account and Treasury Account categories in the Agency's appropriations for a single fiscal year
+
+## GET
+
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "number"
+            }
+    + Parameters
+        + `toptier_code`: 012 (required, number)
+            The toptier code of an agency (could be a CGAG or FREC) so only numeric character strings of length 3-4 are accepted.
+        + `fiscal_year` (optional, number)
+            The desired appropriations fiscal year. Defaults to the current FY
+
++ Response 200 (application/json)
+    + Attributes
+        + `federal_account_count` (required, number)
```

Comment:
@kbard  added the agency code and fiscal year to the response in his PR. Might as well include more details